docs/content/users/quickstart.md

```diff
@@ -509,7 +509,7 @@ The Laravel project type can be used for [StarterKits](https://laravel.com/docs/
 
     ```bash
     mkdir my-laravel-site && cd my-laravel-site
-    # If you're using  an older version of DDEV (<=v1.24.7),
+    # If you're using an older version of DDEV (<=v1.24.7),
```

Comment:
I'm reluctant to add version specs to the docs that get out of date so quickly. I wonder if there's another technique to work on this problem.

Reply:
Another way is not to update the quickstart, as we did with `ddev composer create-project`.
Wait a few months and only do it later.

Reply:
Thank you.